### DISCUSSION TITLE
Tv and hi-fi (audio) icon

### DISCUSSION THREAD

**Discussion by lukaszwitkowski** [RavenSystem/esp-homekit-devices#1632] [MEPLHAA Scripts]
Hello 
Raven have code to remote tv by ir, but i want to turn on/off my tv by relay can i have tv icon and use relay?
The same i need in audio - plug.
On my photo i have original apple tv and original airport2 airplay2
![image](https://user-images.githubusercontent.com/36861888/142181288-9f97d841-df8f-4465-b370-555ac07d3ecb.jpeg)

**Comment by lukaszwitkowski** [comment]
I want that icons and i want to relay not ir ![image](https://user-images.githubusercontent.com/36861888/142182578-6154c95e-8e54-441c-a102-fa9273391d87.jpeg)

**Reply by pawelosin** [reply]
Co do ikony TV, to wystarczy zmienić w konfiguracji podstawowej włącznika "t":1 na "t":60, np:
{
  "c": { "l": 13, "b": [{ "g": 0, "t": 5 }] },
  "a": [{
    "t": 60,
    "0": { "r": [{ "g": 12, "v": 0 }] },
    "1": { "r": [{ "g": 12, "v": 1 }] },
    "b": [{ "g": 0 }]
  }]
}
natomiast co do ikony amplitunera, to chyba nie ma możliwości jej ustawić na razie.
